WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -149,6 +150,24 @@ protected void onNewIntent(Intent intent) {
     private void launchWithNoteId() {
         if (isFinishing() || getIntent() == null) return;
 
+        // Check for push authorization request
+        if (getIntent().hasExtra(NotificationsUtils.ARG_PUSH_AUTH_TOKEN)) {
+            Bundle extras = getIntent().getExtras();
+            String token = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_TOKEN, "");
+            String title = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_TITLE, "");
+            String message = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_MESSAGE, "");
+            long expires = extras.getLong(NotificationsUtils.ARG_PUSH_AUTH_EXPIRES, 0);
```

Comment:
Should the default expiration if none is passed both here and [here](https://github.com/wordpress-mobile/WordPress-Android/pull/2788/files#diff-6493e140350cb7e12c1195918fd6ebd8R284) really be `0`?

My concern is that for some reason the backend changes and `expires` is no longer passed, which would mean the app would always show the toast telling the user they took too long.